src/notifications/changing_strings.md

```diff
@@ -0,0 +1,31 @@
+# Changing default strings
+
+The Notification Plugin uses a set of identifiers and default strings to display messages when processing push notifications. These default strings can be customised by the application. For example, if you wanted to change the `lnurl_pay_metadata_plain_text`, which sets the LNURL-pay text/plain metadata.
```

Comment:
It looks like we're showing how to modify `lnurl_pay_metadata_plain_text` on the following examples. Perhaps it's better to word this as:
```suggestion
The Notification Plugin uses a set of identifiers and default strings to display messages when processing push notifications. These default strings can be customised by the application. For example, to change the `lnurl_pay_metadata_plain_text`, which sets the LNURL-pay text/plain metadata, you can modify it as follows:
```